src/ShallowTraversal.js

```diff
@@ -66,6 +69,18 @@ export function nodeHasId(node, id) {
   return propsOfNode(node).id === id;
 }
 
+
+export function nodeHasProperty(node, propKey, propValue) {
+  const nodeProps = propsOfNode(node);
+
+  if (propValue) {
+    return nodeProps[propKey] === propValue;
+  }
+
+  return nodeProps.hasOwnProperty(propKey);
```

Comment:
I haven't ran into a situation where `hasOwnProperty` didn't work. Can you think of a test case?